gulpfile.js

```diff
@@ -81,6 +83,22 @@ var DEFINES = {
   PDFJS_NEXT: false,
 };
 
+function safeSpawnSync(command, parameters, options) {
+  // Windows has problems executing commands outside a shell.
+  if (process.platform === 'win32') {
```

Comment:
The shell option does work on Linux as well after adjusting the `stdio` behavior, so we can indeed enable it on all platforms to not make it Windows-specific anymore. Done in the new commit.